packages/utils-telemetry/src/telemetry-api.ts

```diff
@@ -3,30 +3,23 @@ type Properties = { [key: string]: string };
 
 type TelemetryItem = {
     data: {
-        baseData: {
-            measurements: Measurements;
-            name: string;
-            properties: Properties;
-            ver?: number;
```

Comment:
Is the version added automatically when sending the telemetry? I know we have a few metrics around this.

Reply:
Is this tracking webhint version? A bunch of other places (including webhint.io) had this hardcoded to 2, so I thought this was the v2 of the app insights REST API.  If you look at Line 84 of this file (before change), you will see if set to 2.

Reply:
TBH, I don't know. @antross it he one that took care of the telemetry and now you so probably you know best how it works 😅

Let's get #3837 merged first and then this one, sounds good?

Reply:
Sounds good ... looks like that one is merged now, is this one good to go then?